rpg-docs/client/views/character/stats/healthCard/healthCard.css

```diff
@@ -3,10 +3,27 @@
 	margin-right: 8px;
 }
 
+.healthCard #temporaryHitPointSlider {
+	--paper-diff-slider-active-color: #009688; /* teal */
+	--paper-diff-slider-knob-color: #009688; /* these don't seem to work? */
+	--paper-diff-slider-pin-color: #009688;
+}
+
+.healthCard .extraHitPointSlider {
+	--paper-diff-slider-active-color: #00BCD4; /* cyan */
+	--paper-diff-slider-knob-color: #00BCD4; /* these don't seem to work? */
```

Comment:
So you can't use custom css attributes (that start with `--`) in css compiled by Meteor. You need to put styles that use custom css attributes in `public > custom_components > app-theme.html`